src/boot.rs

```diff
@@ -181,6 +184,8 @@ pub async fn create_context<H: Hooks>(environment: &str) -> Result<AppContext> {
     let environment = Environment::from_str(environment)
         .unwrap_or_else(|_| Environment::Any(environment.to_string()));
     let config = environment.load()?;
+
+    #[cfg(not(feature = "testing"))]
```

Comment:
I think we can just set "testing = false" in config for test by default,
but then, we need to find the right point to do init in tests only once 

The challenge is finding the right spot to do init only once for the lifetype of a test -- maybe lazy_static

Reply:
Yes i agree, i have more problems when did it like `dead_code` in clippy ignoring. 

this is why i move it to WIP